# HG changeset patch
# User Sukhbir Singh <sukhbir@torproject.org>
# Date 1413962210 -19800
#      Wed Oct 22 12:46:50 2014 +0530
# Node ID ce691e5400583295a0cc0d0d43a3382a2f95940f
# Parent  1172d3f8ed1dd3d7d6f892f4d123b109499e24d3
Ensure TLS/SSL is used by default for Facebook accounts

diff --git a/chat/protocols/facebook/facebook.js b/chat/protocols/facebook/facebook.js
--- a/chat/protocols/facebook/facebook.js
+++ b/chat/protocols/facebook/facebook.js
@@ -38,7 +38,7 @@
     let server = "chat.facebook.com";
     if (this.prefs.prefHasUserValue("server"))
       server = this.getString("server");
-    this._connection = new XMPPSession(server, 5222, "opportunistic_tls",
+    this._connection = new XMPPSession(server, 5222, "require_tls",
                                        this._jid, this.imAccount.password,
                                        this);
   }
